tox.ini

```diff
@@ -41,6 +41,10 @@ commands =
     lint: yamllint -s test/ molecule/
 whitelist_externals =
     find
+# sitepackages=true <- template module <- libselinux bindings which cannot be
+# be installed with pip. Also this lowers virtualenvs footprints and install
+# time
+sitepackages=true
```

Comment:
I blocked this change earlier if you remember.

Can we just recommend using `tox --sitepackages` to folks who want this?

Reply:
@webknjaz adding new guidelines and recomandation would play against making the experience slimless, it would only make it worse. Even those (few) reading the docs would forget to add the extra parameters and run into troubles.

If we are not careful on to easy the first time experience, we may endup flooded by recurring newbe questions, and tickets.